apps/studio/src/schemas/site.ts

```diff
@@ -4,6 +4,11 @@ export const getConfigSchema = z.object({
   id: z.number().min(1),
 })
 
+export const getLocalisedSitemapSchema = z.object({
+  siteId: z.number().min(1),
+  resourceId: z.number().min(1).optional(),
```

Comment:
i think we shuold actually put these as strings because we don't want to treat them as numbers and we don't actually do any numerical operations on them...

having said that, this isn't a big deal because this inconsistency is littered throughout our codebase so we should fix it in 1 go